src/aiortc/contrib/media.py

```diff
@@ -664,7 +666,7 @@ async def __run_track(self, track, context):
                     keypoints_file.write("\n")
                     keypoints_file.close()
 
-                if self.__enable_prediction and not self.__video_queue.empty():
```

Comment:
why remove `self.__video_queue.empty()`?

Reply:
Because once you dequeue the last frame, it stops predictions altogether because the queue is empty even though you have the last reference frame. There's another part of the code that updates the reference frame as needed by waiting on the queue and that takes care of this.